core/src/impl/Kokkos_Profiling.cpp

```diff
@@ -542,6 +550,21 @@ void finalize() {
 #endif
 }
 
+void syncDualView(const std::string& label, const void* const ptr,
+                  bool on_device) {
+  if (Experimental::current_callbacks.sync_dual_view != nullptr) {
+    (*Experimental::current_callbacks.sync_dual_view)(label.c_str(), ptr,
+                                                      on_device);
+  }
+}
+void modifyDualView(const std::string& label, const void* const ptr,
+                    bool on_device) {
+  if (Experimental::current_callbacks.sync_dual_view != nullptr) {
```

Comment:
I think you mean `if (Experimental::current_callbacks.modify_dual_view != nullptr) {`

Reply:
That would have been a very annoying bug. 🍻